practice/src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/main.css">
+  <title>Products Management</title>
+</head>
+
+<body>
+  <!-- Edit Feature -->
+  <div class="form-wrapper">
+    <h2 class="secondary-heading form-heading-confirm">Edit</h2>
+    <form action="javascript:void(0)" class="form-modal">
+      <div class="form-info">
+        <div class="form-group">
+          <label for="editName" class="main-title">Name</label>
+          <input type="text" class="input-form" required />
```

Comment:
Don't use `required` attribute in HTML to validate. Please remove it. We will use JS to validate form